matrix/lambdas/api/core.py

```diff
@@ -154,3 +154,21 @@ def get_matrix(request_id: str):
 def get_formats():
     return ConnexionResponse(status_code=requests.codes.ok,
                              body=[item.value for item in MatrixFormat])
+
+
+def dss_notifications(body):
+    bundle_uuid = body['match']['bundle_uuid']
+    bundle_version = body['match']['bundle_version']
+    subscription_id = body['subscription_id']
+    event_type = body['event_type']
+
+    payload = {
+        'bundle_uuid': bundle_uuid,
+        'bundle_version': bundle_version,
+        'event_type': event_type,
+    }
+    lambda_handler.invoke(LambdaName.NOTIFICATIONS, payload)
```

Comment:
lets consider putting sqs here instead of directly invoking the lambda. There are many reasons to do this 1) Lambda api fails intermittently. Good to have lambda retry driven by sqs 2) control processing rate of notifications 3) track notifications received and failed potentially in a deadletter queue

Reply:
Agreed: https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/matrix-service/227